docs/IE_DG/Int8Inference.md

```diff
@@ -65,44 +23,38 @@ A lot of investigation was made in the field of deep learning with the idea of u
 
 8-bit computations (referred to as `int8`) offer better performance compared to the results of inference in higher precision (for example, `fp32`), because they allow loading more data into a single processor instruction. Usually the cost for significant boost is a reduced accuracy. However, it is proved that an accuracy drop can be negligible and depends on task requirements, so that the application engineer can set up the maximum accuracy drop that is acceptable.
 
-Current Inference Engine solution for low-precision inference uses Intel MKL-DNN and supports inference of the following layers in 8-bit integer computation mode:
-* Convolution
-* FullyConnected
-* ReLU
-* ReLU6
-* Reshape
-* Permute
-* Pooling
-* Squeeze
-* Eltwise
-* Concat
-* Resample
-* MVN
-
-This means that 8-bit inference can only be performed with the CPU plugin on the layers listed above. All other layers are executed in the format supported by the CPU plugin: 32-bit floating point format (`fp32`).
+
+OpenVINO™ runtime supports wide range of INT8 operations. Let's explore quantized [TensorFlow* implementation of ResNet-50](https://github.com/openvinotoolkit/open_model_zoo/tree/master/models/public/resnet-50-tf) model. Use [Model Downloader](https://github.com/openvinotoolkit/open_model_zoo/tree/master/tools/downloader) tool to download the model from [OpenVINO™ Toolkit - Open Model Zoo repository](https://github.com/openvinotoolkit/open_model_zoo):
+```sh
+./downloader.py --name resnet-50-tf --precisions FP16-INT8
```

Comment:
By this command it just downloads resnet_v1-50.pb file.
I would suggest to refer https://github.com/openvinotoolkit/open_model_zoo/tree/develop/tools/downloader readme which already contains instructions for model quantization (quantizer.py) 
Also we need to refer POT and NNCF instruction for model quantization.